src/components/base/Checkbox/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { COLORS } from '@utils/constants';
+import styled from 'styled-components';
+
+const CheckBox = ({id, name, onChange, ...props}) => {
+  const [ isChecked, setChecked ] = useState(false);
+
+  const checkHandler = ({target}) => {
+    setChecked(prev => !prev);
+  }
```

Comment:
```suggestion
  const checkHandler = (e) => {
    setChecked(prev => !prev);
    onChange && onChange(e);
  }
```
onChange가 사용되지 않고 있어서 이렇게 해줘야 할 것 같아요!